packages/nimble-components/src/table/index.ts

```diff
@@ -341,6 +341,7 @@ export class Table<
         this.keyboardNavigationManager.connect();
         document.addEventListener('keydown', this.onKeyDown);
         document.addEventListener('keyup', this.onKeyUp);
+        document.addEventListener('focusout', this.onFocusOut);
```

Comment:
focusout is a bubbling event so this should get triggered if any child has a focusout (and the event bubble up): https://developer.mozilla.org/en-US/docs/Web/API/Element/focusout_event 

Which from the PR description I don't think is the intention. Sounds like the intention is to be tracking the window focus. If that's the case I think we should listen for the non-bubbling `blur` event instead.

Reply:
Good catch. Looks like the document does not get a `blur` event, but interestingly, the window does. I think that's good enough for our purposes. Might be some corner cases with tables inside tabs or iframes, but I'm not sure how we can do better.